### PR TITLE
Replaced the lorem ipsum strings with real text

### PR DESCRIPTION
**Pull request by nllz**

### CODE REVIEW

File: images/dashboard/dashboard.py [hunk 1]
```diff
@@ -93,31 +93,32 @@ def plot_interactions(archive_select):
...
+You can also run these analyses, and many more, by locally installing [BigBang](https://github.com/datactive/bigbang)."
     ),
     archive_select_widget,
 )
 
 plot_daily_activity_boxed = pn.Column(
     pn.pane.Markdown(
-        "#### Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse bibendum, \
-                                                       lectus ut suscipit efficitur, orci erat gravida ligula, sed vestibulum nisi dui nec lorem."
+        "#### In this graph you see the activity over the lifespan of the mailinglist From this you can understand for how long the Working Group exists and how active it has been, and whether there have been any peeks in activity."
```
Comment (sbenthall) [inline images/dashboard/dashboard.py]: missing full stop "." between 'mailinglist' and 'From'

File: images/dashboard/dashboard.py [hunk 2]
```diff
@@ -93,31 +93,32 @@ def plot_interactions(archive_select):
...
     ),
     plot_daily_activity,
 )
 
 get_top_senders_boxed = pn.Column(
     pn.pane.Markdown(
-        "#### Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse bibendum, \
-                                                       lectus ut suscipit efficitur, orci erat gravida ligula, sed vestibulum nisi dui nec lorem."
+        "#### This table shows the information of the top senders to the mailinglist, such as their name, their emailaddress, and the amount of email they have sent."
```
Comment (sbenthall) [inline images/dashboard/dashboard.py]: space in 'emailaddress' please

File: images/dashboard/dashboard.py [hunk 3]
```diff
@@ -93,31 +93,32 @@ def plot_interactions(archive_select):
...
     ),
     get_top_senders,
 )
 
 plot_interactions_boxed = pn.Column(
     pn.pane.Markdown(
-        "#### Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse bibendum, \
-                                                       lectus ut suscipit efficitur, orci erat gravida ligula, sed vestibulum nisi dui nec lorem."
+        "#### This graph show the interaction among the post posters to the working group mailinglist. It allows you to understand who sents the most mails, and to whom they are responding. The bigger the size of the node, the more emails they have sent. If you click on the node, you will the emailaddress of the sender and the amount of emails they have sent"
```
Comment (sbenthall) [inline images/dashboard/dashboard.py]: This graph show**s**

Comment (sbenthall) [reply images/dashboard/dashboard.py]: mailinglist -> mailing list

'you will the emailadrdress' -> you will see the email address

File: images/dashboard/dashboard.py (hunk 2 above)
Comment (nllz) [reply images/dashboard/dashboard.py]: Fixed!

File: images/dashboard/dashboard.py (hunk 3 above)
Comment (nllz) [reply images/dashboard/dashboard.py]: Fixed!

**Pull request merged**